R/cran-compliance.R

```diff
@@ -25,12 +25,12 @@ vendor_pkgs <- function(path = ".", quiet = FALSE, overwrite = NULL) {
   local_quiet_cli(quiet)
 
   # get path to rust folder
-  src_dir <- rprojroot::find_package_root_file(path, "src/rust")
+  src_dir <- rprojroot::find_package_root_file("src", "rust", path = path)
```

Comment:
This is `find_extendr_crate()`. The error message here is better though, maybe we consider updating both `find_extendr_*` functions to use it? Also, should we wrap all `find_package_root_file()` calls in `rlang::try_fetch()`?

Reply:
This call doesn't cause an error if the path doesn't exist which is why the below check exists. 

It may make sense to update `find_extendr_crate()` later on, but I think it is out of scope of this PR. 

As an aside I think the `path` is not set appropriately elsewhere. See how here it was modified to setting the `path` argument?

Reply:
It was passed through dots but it should not have see: 

```r
args(rprojroot::find_package_root_file)
#> function (..., path = ".") 
#> NULL
```

That sounds good, but I still think it is out of scope of this PR since this PR is focused on increasing test coverage. This change was made to improve test coverage. The code below it was never reached because of the misordered arguments